app/static/css/travelForm.css

```diff
@@ -0,0 +1,38 @@
+
+html * {
+    font-size: 13px;
```

Comment:
idk if your chad supervisors care but avoid using px for font-size. it poses a problem for accessibility. when a user changes their browser default font size, rem and em values change accordingly but px values stay the same

here's a reference. https://joshcollinsworth.com/blog/never-use-px-for-font-size